stdlib/crypto/src/main/ballerina/crypto/natives.bal

```diff
@@ -40,10 +40,11 @@ documentation {
 
     P{{baseString}} The string to be hashed
     P{{keyString}} The key string
+    P{{keyEncoding}} The encoding of the key
     P{{algorithm}} The hashing algorithm to be used
     R{{}} The hashed string
 }
-public extern function hmac(string baseString, string keyString, Algorithm algorithm) returns (string);
+public extern function hmac(string baseString, string keyString, string keyEncoding = "UTF-8", Algorithm algorithm) returns (string);
```

Comment:
We have a set of predefined key encoding types. So shall we use a type in crypto package instead of the string here?